cmd/loom/sendtx.go

```diff
@@ -50,16 +55,30 @@ func newMigrationCommand() *cobra.Command {
 			if callerChainID == "" {
 				callerChainID = cli.TxFlags.ChainID
 			}
-			return migrationTx(Id, cli.TxFlags.PrivFile, cli.TxFlags.Algo, callerChainID)
+			if Id == 2 {
```

Comment:
Shift this thing into another method. And use `switch`, so we can add parameter for other migrations too.

Reply:
Done.